src/app/core/entity/entity-actions/entity-bulk-edit/entity-bulk-edit.component.html

```diff
@@ -0,0 +1,33 @@
+<h2 mat-dialog-title i18n>Bulk edit {{ selectedRows }} tasks</h2>
```

Comment:
try using entityConstructor.labelPlural here instead of calling it "tasks" generically?